guessTheNumber.js

```diff
@@ -0,0 +1,19 @@
+var playerNumber = prompt("I'm thinking of a number between 1 and 100. Can you guess the number I'm thinking of?")
+
+min = 1
+max = 100
+var randomize =  Math.random() * (max-min) + min;
+var ourNumber  = Math.round(randomize)
+
+while(ourNumber >= max) {
+   ourNumber--
+}
+while(ourNumber == min) {
+   ourNumber++
+}
+
+if (playerNumber == ourNumber) {
```

Comment:
Best practice is to use "threequals" (`===`) for comparison.  `==` is subject to possible bugs because of implicit type coercion.  `===` checks that both the type and value are the same.